front/src/common/Map/Layers/InfraObjectLayers/InfraObjectLayers.tsx

```diff
@@ -18,84 +23,104 @@ type InfraObjectLayersProps = {
+        layerOrder={LAYER_GROUPS_ORDER[LAYERS.ELECTRIFICATIONS.GROUP]}
+        infraID={infraId}
+        layersSettings={layersSettings}
+      />
 
-    <BufferStops
-      colors={colors[mapStyle]}
-      layerOrder={LAYER_GROUPS_ORDER[LAYERS.BUFFER_STOPS.GROUP]}
-      infraID={infraId}
-    />
+      <NeutralSections
+        colors={colors[mapStyle]}
+        layerOrder={LAYER_GROUPS_ORDER[LAYERS.DEAD_SECTIONS.GROUP]}
+        infraID={infraId}
+        layersSettings={layersSettings}
+        overrideStore={compact}
+      />
 
-    <Detectors
-      colors={colors[mapStyle]}
-      layerOrder={LAYER_GROUPS_ORDER[LAYERS.DETECTORS.GROUP]}
-      infraID={infraId}
-    />
+      <BufferStops
+        colors={colors[mapStyle]}
+        layerOrder={LAYER_GROUPS_ORDER[LAYERS.BUFFER_STOPS.GROUP]}
+        infraID={infraId}
+        layersSettings={layersSettings}
+      />
 
-    <Switches
-      colors={colors[mapStyle]}
-      layerOrder={LAYER_GROUPS_ORDER[LAYERS.SWITCHES.GROUP]}
-      infraID={infraId}
-    />
+      <Detectors
+        colors={colors[mapStyle]}
+        layerOrder={LAYER_GROUPS_ORDER[LAYERS.DETECTORS.GROUP]}
+        infraID={infraId}
+        layersSettings={layersSettings}
+      />
 
-    <SpeedLimits
-      colors={colors[mapStyle]}
-      layerOrder={LAYER_GROUPS_ORDER[LAYERS.SPEED_LIMITS.GROUP]}
-      punctualLayerOrder={LAYER_GROUPS_ORDER[LAYERS.SPEED_LIMITS_PUNCTUAL.GROUP]}
-      infraID={infraId}
-    />
+      <Switches
+        colors={colors[mapStyle]}
+        layerOrder={LAYER_GROUPS_ORDER[LAYERS.SWITCHES.GROUP]}
+        infraID={infraId}
+        layersSettings={layersSettings}
+      />
 
-    <SNCF_PSL
-      colors={colors[mapStyle]}
-      layerOrder={LAYER_GROUPS_ORDER[LAYERS.SPEED_LIMITS.GROUP]}
-      punctualLayerOrder={LAYER_GROUPS_ORDER[LAYERS.SPEED_LIMITS_PUNCTUAL.GROUP]}
-      infraID={infraId}
-    />
+      <SpeedLimits
+        colors={colors[mapStyle]}
+        layerOrder={LAYER_GROUPS_ORDER[LAYERS.SPEED_LIMITS.GROUP]}
+        punctualLayerOrder={LAYER_GROUPS_ORDER[LAYERS.SPEED_LIMITS_PUNCTUAL.GROUP]}
+        infraID={infraId}
+        layersSettings={layersSettings}
+      />
 
-    <Signals
-      sourceTable="signals"
-      colors={colors[mapStyle]}
-      layerOrder={LAYER_GROUPS_ORDER[LAYERS.SIGNALS.GROUP]}
-      infraID={infraId}
-    />
-  </>
-);
+      <SNCF_PSL
+        colors={colors[mapStyle]}
+        layerOrder={LAYER_GROUPS_ORDER[LAYERS.SPEED_LIMITS.GROUP]}
+        punctualLayerOrder={LAYER_GROUPS_ORDER[LAYERS.SPEED_LIMITS_PUNCTUAL.GROUP]}
+        infraID={infraId}
+        layersSettings={layersSettings}
+      />
 
+      <Signals
+        sourceTable="signals"
+        colors={colors[mapStyle]}
+        layerOrder={LAYER_GROUPS_ORDER[LAYERS.SIGNALS.GROUP]}
+        infraID={infraId}
+        layersSettings={layersSettings}
+      />
+      <Platforms
+        colors={colors[mapStyle]}
+        layerOrder={LAYER_GROUPS_ORDER[LAYERS.PLATFORMS.GROUP]}
+      />
```

Comment:
Platforms corresponds to "Quais de gare", the data does not come from our Gaia infra but from OSM, so it should not be here
```suggestion
```